src/System Application/Test/Data Classification/src/DataClassificationMgtTests.Codeunit.al

```diff
@@ -872,8 +860,24 @@ codeunit 135150 "Data Classification Mgt. Tests"
         if FieldsSyncStatus.IsEmpty() then
             FieldsSyncStatus.Insert();
 
-        DataClassificationMgt.InsertDataPrivacyEntity(DataPrivacyEntities, Database::"Fields Sync Status",
-          Page::"Field Content Buffer", 1, '', 1);
+        DataClassificationMgt.InsertDataPrivacyEntity(DataPrivacyEntities, Database::"Fields Sync Status", Page::"Field Content Buffer", 1, '', 1);
+    end;
+
+    local procedure GetNumberOfEnabledSensitiveFieldsForAllSupportedTables() FieldCount: Integer
+    var
+        TableMetaData: Record "Table Metadata";
+        Field: Record Field;
+        DataClassificationMgtImpl: Codeunit "Data Classification Mgt. Impl.";
+    begin
+        FieldCount := 0;
+        if TableMetaData.FindSet() then
+            repeat
+                if DataClassificationMgt.IsSupportedTable(TableMetaData.ID) then begin
+                    Field.SetRange(TableNo, TableMetaData.ID);
+                    DataClassificationMgtImpl.GetEnabledSensitiveFields(Field);
```

Comment:
use of internals .. tsk tsk... should use a test library...

Reply:
Good catch will move it after I fix all the tests...